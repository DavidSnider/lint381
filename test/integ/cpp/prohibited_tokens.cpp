memset(foo, bar);
memmove(baz, qux);
memcpy(grault, xyzzy);
exit(1);
