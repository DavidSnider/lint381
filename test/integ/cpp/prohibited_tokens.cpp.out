prohibited_tokens.cpp:1:1: error: Don't use 'memset'
memset(foo, bar);
^^^^^^
prohibited_tokens.cpp:2:1: error: Don't use 'memmove'
memmove(baz, qux);
^^^^^^^
prohibited_tokens.cpp:3:1: error: Don't use 'memcpy'
memcpy(grault, xyzzy);
^^^^^^
prohibited_tokens.cpp:4:1: error: Don't use 'exit'
exit(1);
^^^^
